### PR TITLE
Grammar fixes to readme.md

### PR DESCRIPTION
**Pull request by LillyWho**
Fixed some phrasing to make the text a little bit more readable.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,13 +16,13 @@ Trolleybus Simulator addon for Garry's Mod.
...
 * Well-developed trolleybuses
-* Contact network system
+* Catenary network system
 * Routes system: trolleybus stops and passengers
 * Simple traffic system
 * Map created for trolleybuses
 * Editor tools for creating infrastructure for trolleybuses
 
-Mechanics of trolleybuses based on real world. Controls contains many subsystems (engine, circuits, informers, control systems, etc). This will make you feel like a real trolleybus driver.
+Mechanics of trolleybuses based on real world. The interactive elements consist of many subsystems (engine, circuits, informers, control systems, etc). This will make you feel like a real trolleybus driver.
```
Comment (ShadowBonnieRUS) [inline README.md:25]: `Controls` means here not just interactive elements, but also technical elements (like engine, circuits).

File: README.md [hunk 2]
```diff
@@ -32,20 +32,20 @@ List of main trolleybuses:
 * AKSM-333 (few mods)
 * TrolZa-5265
 
-Contact network is strongly detailed. It contains main elements like wires, switches, crosses and etc. In trolleybuses and contact network used electric circuits computations to simulate real world events.
+The catenary network is strongly detailed. It contains main elements like wires, switches, crosses and etc. The trolleybuses and catenary network use electric circuit computations to simulate real world events.
```
Comment (ShadowBonnieRUS) [inline README.md:35]: Is `The` in `The trolleybuses` redundant?

File: README.md [hunk 3]
```diff
@@ -32,20 +32,20 @@ List of main trolleybuses:
...
-* Contact network editor (for creating contant network on map)
-* Routes editor (for creating trolleybus stops and routes to use with)
-* Traffic tracks editor (for creating data used by AI traffic cars to navigate around map)
+Players can use built-in tools on any map to create trolleybus infrastructure. Provided in-game editors are:
+* Catenary network editor (for creating catenary network on map)
+* Routes editor (for creating trolleybus stops and routes)
+* Traffic tracks editor (for creating data used by AI traffic cars to navigate around the map)
 * Traffic lights editor (for creating traffic lights in order to comply with traffic rules by trolleybuses and AI traffic cars)
-* Informers editor (for creating informer sound lines used in trolleybus informers)
+* Announcer editor (for creating announcer sound lines used in trolleybus announcements)
```
Comment (ShadowBonnieRUS) [inline README.md:46]: Also all other `informer`s across document can be changed to `announcer`.

File: README.md (hunk 1 above)
Comment (LillyWho) [reply README.md:25]: Come to think of it, I'd change it to "The simulated elements" instead.

**Comment by LillyWho** [comment]
I've updated as suggested

**Pull request merged**